feedthefox/dashboard/templates/jinja2/dashboard.html

```diff
@@ -24,29 +26,43 @@
             <span class="caret"></span>
           </button>
           <ul class="dropdown-menu text-left" aria-labelledby="reportmenu">
-            <li><a href="#">Request Feature</a></li>
-            <li><a href="#">Report Defect</a></li>
+            <li><a href="https://bugzilla.mozilla.org/form.fxos.feature" target="_blank">Request Feature</a></li>
+            <li><a href="https://bugzilla.mozilla.org/form.fxos.feature?feature_type=Existing%20Feature%20Defect" target="_blank">Report Defect</a></li>
           </ul>
         </div>
       </div>
     </div>
 
-    <table class="table table-striped">
+    <table class="bug-list table table-bordered table-responsive sortable">
       <thead>
         <tr>
-          <th>Bug</th>
-          <th>Summary</th>
-          <th>Votes</th>
+          <th class="hidden-xs">Bug</th>
```

Comment:
Please use the same alignment used in ``<tbody>``.
* ``Bug``, ``Votes``: ``.text-center``